package.json

```diff
@@ -10,6 +10,7 @@
     "start-teleport-e": "yarn workspace @gravitational/teleport.e start",
     "build-term": "yarn workspace @gravitational/teleterm build",
     "start-term": "yarn workspace @gravitational/teleterm start",
+    "dev-term": "yarn workspace @gravitational/teleterm dev",
```

Comment:
Initially we added native deps to start-term after the hackathon because Krzysztof and Edoardo had a ton of problems. They removed the whole node_modules each time they ran into some kind of a problem with JS and then they didn't remember about reinstalling native deps. ;)

So we just put native deps there. Grzegorz knew that it'll be slow as hell on Windows, but I was like "We'll deal with that later".

Well, last week I had to finally build the app on Windows and it was slow as hell. So it's good that you added it there because I'd probably end up doing the same in a week or two.